t/smoke/tasks.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import os
+import sys
+from signal import SIGKILL
+from time import sleep
+
+import celery.utils
+from celery import Task, shared_task, signature
+from celery.canvas import Signature
+from t.integration.tasks import *  # noqa
```

Comment:
Why?

Reply:
To allow the smoke tests containers access to all of the codebase of the integration tests tasks.
This will greatly reduce code duplication